### PR TITLE
Update Coral

### PR DESCRIPTION
**Pull request by ksobolew**
## Description

The previous version of Coral was about 11 months old by now. And Ivy 2.4.0 it depends on is also outdated.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1709,6 +1709,13 @@
                 <version>${dep.antlr.version}</version>
             </dependency>
 
+            <dependency>
+                <!-- Remove when Coral catches up with its transitive dependency version -->
+                <groupId>org.apache.ivy</groupId>
+                <artifactId>ivy</artifactId>
+                <version>2.5.1</version>
```
Comment (martint) [inline pom.xml:1716]: What’s the issue with the version Coral depends on?

Comment (ksobolew) [reply pom.xml:1716]: If you guessed "security scanners" then you guessed right. The most likely outcome here is just that this is a false positive[^1], but I'm not very familiar with how this is used in Trino so I'm currently exploring possibilities here.

[^1]: It's used for one very specific thing which does not handle user-provided input.

Comment (ksobolew) [reply pom.xml]: This has been pushed upstream and is no longer necessary.

**Review by kokosing — APPROVED** [review]

**Comment by ksobolew** [comment]
Timeout in Cassandra product tests:

```
tests               | 2023-04-20 03:18:48 INFO: FAILURE     /    io.trino.tests.product.cassandra.TestSelect.testSelectAllTypePartitioningMaterializedView (Groups: profile_specific_tests, cassandra) took 13.4 seconds
tests               | 2023-04-20 03:18:48 SEVERE: Failure cause:
tests               | com.datastax.oss.driver.api.core.DriverTimeoutException: Query timed out after PT2S
```

**Pull request merged**